opm/simulators/flow/FlowGenericVanguard.cpp

```diff
@@ -175,6 +175,34 @@ std::string FlowGenericVanguard::canonicalDeckPath(const std::string& caseName)
     throw std::invalid_argument("Cannot find input case '"+caseName+"'");
 }
 
+void FlowGenericVanguard::updateNOSIM_(std::string_view dryRunString)
+{
+    try {
```

Comment:
broken indentation. sorry for nitpicking.

Reply:
Thanks for the review and sorry for the broken indentation.